web/src/components/studies/CreateStudyForm.tsx

```diff
@@ -0,0 +1,333 @@
+  dataProtectionNumber: number;
+  nhsEnglandRef: number;
+  irasId: string; // might be number, unclear
+  // checkboxes
+  uclSponsorship: boolean;
+  cag: boolean;
+  ethics: boolean;
+  hra: boolean;
+  secureData: boolean;
+  redcap: boolean;
+  tre: boolean;
+  dbs: boolean;
+  dataProtection: boolean;
+  thirdParty: boolean;
+  externalUsers: boolean;
+  consent: boolean;
+  nonConsent: boolean;
+  extEea: boolean;
+  nhs: boolean;
+  nhsEngland: boolean;
+  mnca: boolean;
+  dspt: boolean;
+}
+export default function CreateStudyForm(CreateStudyProps: CreateStudyProps) {
+  const { username, setCreateStudyFormOpen } = CreateStudyProps;
+  const {
+    register,
+    handleSubmit,
+    control,
+    formState: { errors, isValid, isSubmitting },
+  } = useForm<CreateStudyValues>({ mode: "onChange", criteriaMode: "all" });
+  const [step, setStep] = useState(1);
+  const nextStep = () => setStep(step + 1);
+  const prevStep = () => setStep(step - 1);
+
+  const ownerValue = useWatch({
+    name: "owner",
+    control,
+  });
+  const ownerIsUser = ownerValue === username;
+  const showCagRef = useWatch({
+    name: "cag",
+    control,
+  });
+  const showIrasId = useWatch({
+    name: "hra",
+    control,
+  });
+  const showDataProtectionNumber = useWatch({
+    name: "dataProtection",
+    control,
+  });
+  const showNhsRelated = useWatch({
+    name: "nhs",
+    control,
+  });
+  const showNhsEnglandRef = useWatch({
+    name: "nhsEngland",
+    control,
+  });
+
+  const onSubmit: SubmitHandler<CreateStudyValues> = (data) => {
+    console.log(data);
+  };
+  return (
+    <Dialog setDialogOpen={setCreateStudyFormOpen} className={styles["study-dialog"]}>
+      <h2>Create Study</h2>
+      <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
+        {step == 1 && (
+          <>
+            <fieldset className={styles.fieldset}>
+              <legend>Study Name</legend>
+              <Label htmlFor="shortStudyName">
+                Short Study Name*:
+                <Input
+                  type="text"
+                  id="shortStudyName"
+                  {...register("shortStudyName", { required: true, maxLength: 55 })}
+                />
+                <HelperText>Maximum 55 characters, do not include any special characters</HelperText>
+                {errors.shortStudyName && <span>This field is required</span>}
+                {/* todo: validation for no special characters */}
+              </Label>
+
+              <Label htmlFor="longStudyName">
+                Long Study Name:
+                <Textarea id="longStudyName" {...register("longStudyName", { maxLength: 255 })} />
+              </Label>
+            </fieldset>
+
+            <fieldset className={styles.fieldset}>
+              <legend>Ownership</legend>
+              <Label htmlFor="owner">
+                Study Owner (PI)*:
+                <Input type="email" id="owner" {...register("owner", { required: true })} />
```

Comment:
I think this should default to the logged-in user

Reply:
Suppose that's related to what we were chatting about last week with possibly auto-approving some based on scores/factors and then higher risk ones can be manually approved